client/signup/steps/theme-selection/index.jsx

```diff
@@ -76,8 +76,7 @@ class ThemeSelectionStep extends Component {
 		const { translate } = this.props;
 		const headerText = translate( 'Choose a theme.' );
 		const subHeaderText = translate(
-			'No need to overthink it. You can always switch to a different theme later.',
-			{ context: 'Themes step subheader in Signup' }
+			'Pick one of our popular themes to get started or choose from hundreds more after you sign up.'
```

Comment:
Hi! I've found a possible matching string that has already been translated **2** times:
`translate( 'Pick one of our popular themes to get started or choose from hundreds more after you sign up.', { context: 'Themes step subheader in Signup'} )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).